packages/ui/src/components/ToastBar.tsx

```diff
@@ -42,15 +45,7 @@ const ToastBar = ({ toast }: Props) => {
           <CloseIcon fontSize="small" />
         </IconButton>
       }
-      message={
-        <ToastContent
-          type={type}
-          title={title}
-          key={id}
-          id={id}
-          link={link}
-        />
-      }
+      message={<ToastContent toast={toast} />}
     />
   )
 }
```

Comment:
we can remove the wrapping part as we have no styles

export default styled(ToastBar)(
  ({ theme }) => `
`
)